newdle/client/src/components/ParticipantTablev2.js

```diff
@@ -0,0 +1,137 @@
+import {
+  getNewdleDuration,
+  getNewdleParticipants,
+  getNewdleTimezone,
+  getNewdleTimeslots,
+} from '../selectors';
+import UserAvatar from './UserAvatar';
+import styles from './creation/userSearch/UserSearch.module.scss';
+import PropTypes from 'prop-types';
+
+function formatMeetingTime(startTime, duration) {
+  const endTime = moment(startTime)
+    .add(duration, 'm')
+    .format('HH:mm');
+  return `${serializeDate(startTime, 'HH:mm')} - ${endTime}`;
+}
+
+function DateCell({timeslot}) {
+  const newdleTimezone = useSelector(getNewdleTimezone);
+  const duration = useSelector(getNewdleDuration);
+
+  const startTime = toMoment(timeslot, 'YYYY-MM-DDTHH:mm');
+  return (
+    <Table.HeaderCell textAlign="center">
+      <div>
+        <div>{startTime.format('D MMM')}</div>
+        <div>{formatMeetingTime(startTime, duration)}</div>
+        <div>({newdleTimezone})</div>
+      </div>
+    </Table.HeaderCell>
+  );
+}
+
+DateCell.propTypes = {
+  timeslot: PropTypes.string,
+};
+
+function ParticipantRow({participant}) {
+  const timeslots = useSelector(getNewdleTimeslots);
+  let participantName = participant.name;
+  let participantEmail = participant.email;
+  // eslint-disable-next-line react/prop-types
+  let matches = participantName.match(/\b(\w)/g);
+  let initials = matches.join('');
+
+  return (
+    <Table.Row textAlign="center">
+      <Table.Cell>
+        <div style={{display: 'flex'}}>
+          <UserAvatar
+            user={{email: participantEmail, name: participantName, initials}}
+            size={25}
+            className={styles['participant-avatar']}
+          />
+          {participantName}
+        </div>
+      </Table.Cell>
+      {timeslots.map((timeslot, index) => {
+        let status = null;
+        if (participant.answers[timeslot] === 'available') {
+          status = 'available';
+        } else if (participant.answers[timeslot] === 'unavailable') {
+          status = 'unavailable';
+        } else if (participant.answers[timeslot] === 'ifneedbe') {
+          status = 'ifneedbe';
+        }
+        return (
+          <Table.Cell key={`${status}-${index}`} textAlign="center">
+            {!!status ? (
+              status === 'available' ? (
+                <Icon color="green" name="checkmark" size="large" />
+              ) : status === 'unavailable' ? (
+                <Icon color="red" name="close" size="large" />
+              ) : (
+                <Icon color="yellow" name="question" size="large" />
+              )
+            ) : null}
+          </Table.Cell>
+        );
+      })}
+    </Table.Row>
+  );
+}
+
+ParticipantRow.propTypes = {
+  participant: PropTypes.arrayOf(
+    PropTypes.shape({
+      answers: PropTypes.objectOf(PropTypes.oneOf(['unavailable', 'available', 'ifneedbe'])),
+      email: PropTypes.string,
+      name: PropTypes.string.isRequired,
+      status: PropTypes.string.isRequired,
+    })
+  ).isRequired,
+};
```

Comment:
```suggestion
ParticipantRow.propTypes = {
  participant: PropTypes.shape({
    answers: PropTypes.objectOf(PropTypes.oneOf(['unavailable', 'available', 'ifneedbe'])),
    email: PropTypes.string,
    name: PropTypes.string.isRequired,
  }).isRequired,
};
```
(status does not seem to be in this witht he nedwles where I tested this)